otherlibs/win32unix/gettimeofday.c

```diff
@@ -20,21 +20,36 @@
 #include "unixsupport.h"
 
 /* Unix epoch as a Windows timestamp in hundreds of ns */
-#define epoch_ft 116444736000000000.0;
+#define epoch_ft 116444736000000000;
 
 CAMLprim value unix_gettimeofday(value unit)
 {
   FILETIME ft;
   double tm;
+  ULARGE_INTEGER li;
   GetSystemTimeAsFileTime(&ft);
-#if defined(_MSC_VER) && _MSC_VER < 1300
-  /* This compiler can't cast uint64_t to double! Fortunately, this doesn't
-     matter since SYSTEMTIME is only ever 63-bit (maximum value 31-Dec-30827
-     23:59:59.999, and it requires some skill to set the clock past 2099!)
-   */
-  tm = *(int64_t *)&ft - epoch_ft; /* shift to Epoch-relative time */
-#else
-  tm = *(uint64_t *)&ft - epoch_ft; /* shift to Epoch-relative time */
-#endif
```

Comment:
This was removed because of this comment on the windows API documentation: 
> Do not cast a pointer to a FILETIME structure to either a ULARGE_INTEGER* or __int64* value because it can cause alignment faults on 64-bit Windows.

Ref: https://docs.microsoft.com/en-us/windows/win32/api/minwinbase/ns-minwinbase-filetime